src/AbuseIpDb.php

```diff
@@ -3,14 +3,17 @@
 namespace nickurt\AbuseIpDb;
 
 use \GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
 use \nickurt\AbuseIpDb\Exception\MalformedURLException;
 
+class AbuseIpDbException extends \Exception{};
```

Comment:
I think it's better to move this to the \Exception folder ..

Reply:
yea, right.